### PR TITLE
[Infra] Split UT cases into CI and nightly for pre-CI speedup

### PR DESCRIPTION
**Pull request by lvliang-intel**
## Type of Change

feature
API not changed

## Description

Split UT cases into CI and nightly for pre-CI speedup

## Expected Behavior & Potential Risk

The UT cases will be covered by CI and nightly. The cost time of pre-CI will be reduced.

## How has this PR been tested?

Local test and pre-CI.

## Dependency Change?

None.

### CODE REVIEW

File: intel_extension_for_transformers/neural_chat/tests/ci/api/test_chatbot_build_api.py [hunk 1]
```diff
@@ -30,24 +30,24 @@ def tearDown(self) -> None:
...
         chatbot = build_chatbot(config)
         self.assertIsNotNone(chatbot)
         response = chatbot.predict(query="Tell me about Intel Xeon Scalable Processors.")
         print(response)
         self.assertIsNotNone(response)
 
     def test_build_chatbot_with_customized_pipelinecfg(self):
-        config = PipelineConfig(model_name_or_path="facebook/opt-125m",
-                                tokenizer_name_or_path="EleutherAI/gpt-neox-20b")
+        config = PipelineConfig(model_name_or_path="/tf_dataset2/models/nlp_toolkit/opt-125m",
```
Comment (hshen14) [inline intel_extension_for_transformers/neural_chat/tests/ci/api/test_chatbot_build_api.py:41]: suggest not using specific path e.g., tf_datasets?

Comment (lvliang-intel) [reply intel_extension_for_transformers/neural_chat/tests/ci/api/test_chatbot_build_api.py:41]: @VincyZhang please check haihao's comments.

Comment (chensuyue) [reply intel_extension_for_transformers/neural_chat/tests/ci/api/test_chatbot_build_api.py]: In order to save test time, we use local model. It's just a folder name created inside the test docker, we can also change to `/datasets`.

**Comment by chensuyue** [comment]
@lvliang-intel the final issue left in this PR, core dumped for this case. 
https://github.com/intel/intel-extension-for-transformers/actions/runs/6747307116/job/18354301432?pr=582#step:8:9360

**Pull request merged**